TestsCalculator/CalculatorPageTests.cs

```diff
@@ -90,13 +320,46 @@ public void PositiveSelectDate()
             percentField.SendKeys("10");
             termField.SendKeys("20");
             dayDropdown.SendKeys("7");
-            monthDropdown.SendKeys("Jun");
+            monthDropdown.SendKeys("Jun"); // why does not fail?
             yearDropdown.SendKeys("2022");
 
             //Assert
+            IWebElement incomeField = driver.FindElement(By.Id("income"));
+            IWebElement interestField = driver.FindElement(By.Id("interest"));
             IWebElement endDateField = driver.FindElement(By.Id("endDate"));
             Assert.AreEqual("27/06/2022", endDateField.GetAttribute("value"));
+            Assert.AreEqual("1005.48", incomeField.GetAttribute("value"));
+            Assert.AreEqual("5.48", interestField.GetAttribute("value"));
+        }
+
+        [Test]
+        public void PositiveSelectPastDate()
+        {
+            //Arrange
+            IWebElement amountField = driver.FindElement(By.Id("amount"));
+            IWebElement percentField = driver.FindElement(By.Id("percent"));
+            IWebElement termField = driver.FindElement(By.Id("term"));
+            IWebElement dayDropdown = driver.FindElement(By.Id("day"));
+            IWebElement monthDropdown = driver.FindElement(By.Id("month"));
+            IWebElement yearDropdown = driver.FindElement(By.Id("year"));
+
+            //Act
+            amountField.SendKeys("1000");
+            percentField.SendKeys("10");
+            termField.SendKeys("20");
+            dayDropdown.SendKeys("21"); // why does not work for 1
+            monthDropdown.SendKeys("March");
+            yearDropdown.SendKeys("2010");
```

Comment:
It's better to work with dropdowns via SelectElement:
```suggestion
            new SelectElement(dayDropdown).SelectByText("21");
            new SelectElement(monthDropdown).SelectByText("March");
            new SelectElement(yearDropdown).SelectByText("2010");
```